app/scripts/navbar/navbar-partial.html

```diff
@@ -10,7 +10,7 @@
                     <span class="icon-bar"></span>
                     <span class="icon-bar"></span>
                 </button>
-                <a class="navbar-brand" href="#">Ashlar Space &amp; Time Database Builder v1.0</a>
+                <a class="navbar-brand" href="#">Grout Space &amp; Time Database Builder v1.0</a>
```

Comment:
Perhaps remove the version number as well? I don't think anyone will want (or remember) to edit this each time we push a new Grout version to PyPi.